src/renderer/store/modules/network.js

```diff
@@ -69,6 +69,27 @@ export default new BaseModule(NetworkModel, {
       commit('SET_ALL', NETWORKS)
     },
 
+    /*
+     * Update data of the network
+     */
+    async updateData ({ commit, rootGetters }, network = null) {
+      if (!network) {
+        network = rootGetters['session/network']
+      }
+
+      try {
+        const crypto = await Client.fetchNetworkCrypto(network.server)
+        const { constants } = await Client.fetchNetworkConfig(network.server)
+        commit('UPDATE', {
+          ...network,
+          crypto,
+          milestone: constants
```

Comment:
The network already has the `constants` field, it can only be updated instead of duplicating the data.

Reply:
Updated this @luciorubeens 	👌